app/views/root/_assisted_digital_help_with_fees_satisfaction_survey.html.erb

```diff
@@ -0,0 +1,57 @@
+<h2 class="satisfaction-survey-heading">Help with fees</h2>
+
+<form class="contact-form" action="/contact/govuk/assisted-digital-help-with-fees-survey-feedback" method="post" id="completed-transaction-form">
+  <input type="hidden" id="service_slug" name="service_feedback[slug]" value="<%= publication.slug.gsub("done/", "") %>" />
+  <input type="hidden" id="service_done_page_url" name="service_feedback[url]" value="<%= publication.web_url %>" />
+
+  <fieldset>
+    <legend><h2>Did you have any assistance with applying for help with fees online today?</h2></legend>
+    <br />
+    <input type="radio" required name="service_feedback[assistance]" id="no-help" value="no">
+    <label for="no-help">No, I applied for help with fees online myself</label>
+    <br />
+    <input type="radio" required name="service_feedback[assistance]" id="some-help" value="some">
+    <label for="no-help">Someone helped me to apply for help with fees online</label>
```

Comment:
Looks like the for attribute is mistakenly duplicated? This will mean in assistive tech like screenreaders all these inputs will be the same.